scripts/obtain/dotnet-install.ps1

```diff
@@ -0,0 +1,338 @@
+#
```

Comment:
no, too many people are using install.ps1 and install.sh so we are leaving two copies of it for now. once we get permanent links for them the content of install.ps1/sh will be replaced with a warning and NZEC so that people can act accordingly and update their script.